tui/src/main.rs

```diff
@@ -7,7 +7,7 @@ pub mod state;
 mod theme;
 
 use std::{
-    io::{self, stdout},
+    io::{self, stdout, Stdout},
```

Comment:
`Stdout` is being used only once.
```suggestion
    io::{self, stdout},
```